raiden/transfer/token_network.py

```diff
@@ -89,8 +89,8 @@ def handle_channelnew(token_network_state, state_change):
     # the ethereum node
     if channel_identifier not in token_network_state.channelidentifiers_to_channels:
         token_network_state.channelidentifiers_to_channels[channel_identifier] = channel_state
-        partneraddresses_to_channels = token_network_state.partneraddresses_to_channels
-        partneraddresses_to_channels[partner_address][channel_identifier] = channel_state
+        addrs_to_ids = token_network_state.partneraddresses_to_channelidentifiers
```

Comment:
nit: two abbreviations in one variable name is a bit too much. Perhaps change to `addresses_to_ids`?